docusaurus/docs/advanced-configuration.md

```diff
@@ -51,18 +49,17 @@ module.exports = {
 
 ---
 
-### Extending the Jest config
+### Extend the Jest config
 
-There are two configuration in the project root that belong to Jest: `jest-setup.js` and `jest.config.js`.
+There are two files in the project root that belong to Jest: `jest-setup.js` and `jest.config.js`.
 
-**`jest-setup.js`:** A file that is run before each test file in the suite is executed. We are using it to
-set up the Jest DOM for the testing library and to apply some polyfills. ([link to Jest docs](https://jestjs.io/docs/configuration#setupfilesafterenv-array))
+**`jest-setup.js`:** Is run before each test file in the suite is executed. It will set up Jest DOM for the testing library and apply some polyfills. ([link to Jest docs](https://jestjs.io/docs/configuration#setupfilesafterenv-array))
```

Comment:
In **`jest-setup.js`:** Is, change ":** Is" to "** is" or revert to "... A file..."